docs/reference/ccr/index.asciidoc

```diff
@@ -307,6 +307,33 @@ and use {ccr}:
 * <<ccr-auto-follow>>
 * <<ccr-upgrading>>
 
+[discrete]
+[[ccr-limitations]]
+=== {ccr-cap} limitations
+{ccr-cap} is designed to replicate user-generated indices only, and doesn't 
+currently replicate any of the following:
+
+* <<system-indices,System indices>>
+* {ml-docs}/machine-learning-intro.html[Machine learning jobs]
+* <<index-templates,index templates>>
+* <<index-lifecycle-management,{ilm-cap}>> and
+<<snapshot-lifecycle-management-api,{slm}>> polices
+* {ref}/mapping-roles.html[User permissions and role mappings]
+* <<snapshots-register-repository,Snapshot repository settings>>
+* <<modules-cluster,Cluster settings>>
+* <<searchable-snapshots,Searchable snapshot>>
+* <<restore-index-data-stream,Restored indices>>
```

Comment:
Is this true, would auto-follow patterns not hit that?

Reply:
This was referring to the problem described in this issue: https://github.com/elastic/elasticsearch/issues/87055

Reply:
Based on the conversation in #87055, it sounds like restored indices aren't replicated if the leader index is deleted and then restored, but the original replicated index still exists in the remote cluster. The restored index in the leader cluster gets a new UUID, so the auto-follow pattern on the remote cluster sees it as new index, but it's actually just a replicated index. 

This sounds like an edge case that we could call attention to, but it seems like restored indices _would be_ replicated if the underlying leader index is still intact. Is that true @henningandersen?

Reply:
I agree @lockewritesdocs, it will follow just fine if the restore results in an unoccupied name on the follower. This is similar to any sort of naming conflict on the follower though a delete and restore on the leader side is possibly a common cause of this.

I'd opt to remove this, since it is not true that restored indices are not followed.

Reply:
👍 Sounds good @henningandersen -- I've removed "Restored indices" from the list of things that aren't followed.